### PR TITLE
Update AboutUs.md and UserGuide.md for v1.1

### PR DESCRIPTION
**Pull request by ErnestCuong**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,51 +9,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Hoang Le Tri Cuong
```
Comment (wpinrui) [inline docs/AboutUs.md:12]: LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -166,6 +166,32 @@ AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.
 If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
 </div>
 
+### Adding progress for a student : `add -p`
```
Comment (wpinrui) [inline docs/UserGuide.md:169]: LGTM

File: docs/team/ernestcuong.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (wpinrui) [inline docs/team/ernestcuong.md:1]: LGTM

**Review by wpinrui — APPROVED** [review]

File: docs/AboutUs.md [hunk 4]
```diff
@@ -9,51 +9,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-
-* Role: Team Lead
-* Responsibilities: UI
-
-### Johnny Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](https://https://github.com/ErnestCuong)]
```
Comment (JunWei3112) [inline docs/AboutUs.md:16]: Should the GitHub address here be `https://github.com/ErnestCuong` instead?

**Review by JunWei3112 — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 4 above)
Comment (ErnestCuong) [reply docs/AboutUs.md:16]: You are correct. My bad. When I copied the address there was no `https://` but when pasting it somehow it appeared.

File: docs/AboutUs.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 <img src="images/ernestcuong.png" width="200px">
 
-[[github](https://https://github.com/ErnestCuong)]
+[[github](https://github.com/ErnestCuong)]
```
Comment (JunWei3112) [inline docs/AboutUs.md:16]: LGTM

**Review by JunWei3112 — APPROVED** [review]

**Pull request merged**